web-console/package.json

```diff
@@ -71,6 +71,7 @@
     "react-redux": "^8.1.2",
     "reactflow": "^11.8.0",
     "ts-pattern": "^5.0.5",
+    "ts-practical-fp": "https://github.com/Karakatiza666/ts-practical-fp",
```

Comment:
can you publish this package on npm? Otherwise we don't have it versioned and if the code in the repo or location of repo changes later we don't have reproducible builds anymore

Reply:
Would you feel comfortable with anchoring to git commit version?

Reply:
I could also add a git tag to look like `"ts-practical-fp": "https://github.com/Karakatiza666/ts-practical-fp#0.0.1"`. I would just prefer to not manage an npm deployment yet

Reply:
Best would be to release on npm. Is it so difficult to release there? This sounds like it's quite a minimal process: https://docs.npmjs.com/creating-and-publishing-scoped-public-packages e.g., just release as v0.0.1 there?

If the `ts-practical-fp` git repo changes in the future where it is renamed or deleted then all revisions in web-console dbsp repo which had this URL will be instantly broken. It's much harder to break when it's on npm.